tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -293,6 +342,60 @@ def test_alice_profiler_user_workflow_single_batch(
     assert not usage_stats_invalid_messages_exist(messages=caplog.messages)
 
 
+@freeze_time(TIMESTAMP)
+def test_alice_expect_column_values_to_match_regex_auto_yes_default_profiler_config_yes_custom_profiler_config_no(
+    alice_validator,
+) -> None:
+    validator = alice_validator
+
+    result = validator.expect_column_values_to_match_regex(
+        column="id",
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+    )
+
+    assert result.success
+
+    expectation_config_kwargs: dict = result.expectation_config.kwargs
+    assert expectation_config_kwargs == {
+        "auto": True,
+        "batch_id": "cf28d8229c247275c8cc0f41b4ceb62d",
+        "column": "id",
+        "include_config": True,
+        "mostly": 1.0,
+        "regex": "(?:[A-Fa-f0-9]){0,4}(?: ?:? ?(?:[A-Fa-f0-9]){0,4}){0,7}",
+        "result_format": "SUMMARY",
+    }
```

Comment:
Missing type hints

Reply:
done! sorry I was being lazy here I appreciate the diligence around type annotating 😅